### PR TITLE
SOFTWARE-5217: Update Major Package Release Policy

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/policy/community-testing.md [hunk 1]
```diff
@@ -29,20 +29,22 @@ After users have been notified of changes, minor packages will be marked eligibl
 negative feedback after 7 calendar days.
 In addition to the above requirements, major packages must also receive positive feedback and be approved by the Release
 Manager.
+If a major package's functionality is tested by the OSG VM Universe tests, the release manager may approve that
+package for release without positive feedback after the package has been in testing for four weeks.
```
Comment (brianhlin) [inline docs/policy/community-testing.md:33]:
```suggestion
If a major package has not received feedback after four weeks and it has been sufficiently tested by OSG integration
tests, the Release Manager may approve the package for release.
```

I think jamming the two conditions together in a single clause makes it easier to understand

File: docs/policy/community-testing.md [hunk 2]
```diff
@@ -29,20 +29,22 @@ After users have been notified of changes, minor packages will be marked eligibl
...
 If a package receives negative feedback, the offending package will be removed from the relevant testing repository.
 
 Major Packages
 --------------
 
 The following packages are considered critical to the production Open Science Grid:
 
-- BLAHP
```
Comment (brianhlin) [inline docs/policy/community-testing.md:39]: Should we keep this in since it's still a separate package in 3.6 release?

**Review by brianhlin — COMMENTED** [review]
Could you also add an update to the version history section below?

**Comment by brianhlin** [comment]
@timtheisen ping on the review comments

File: docs/policy/community-testing.md (hunk 2 above)
Comment (timtheisen) [reply docs/policy/community-testing.md:39]: I can go either way on this. We don't plan any more blahp releases and the integrated blahp is in wide used.

Comment (timtheisen) [reply docs/policy/community-testing.md]: Restored

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**